### PR TITLE
Material upload

### PR DESCRIPTION
**Pull request by bandvov**

### CODE REVIEW

File: modules/material/material.service.js [hunk 1]
```diff
@@ -32,11 +36,46 @@ class MaterialsService {
     return await Material.findByIdAndUpdate(id, material, { new: true });
   }
 
-  async addMaterial(data) {
-    if (await this.checkMaterialExist(data)) {
+  async addMaterial({ material, images }) {
+    const { additionalPrice, ...rest } = material;
+
+    const currency = await Currency.findOne();
```
Comment (dimaPalii) [inline modules/material/material.service.js:42]: Place, move currency declaration after validation checks.
There is no need in 'currency' const if material exist or no images.

File: modules/material/material.service.js [hunk 2]
```diff
@@ -32,11 +36,46 @@ class MaterialsService {
...
+
+    if (await this.checkMaterialExist(rest)) {
```
Comment (dimaPalii) [inline modules/material/material.service.js:44]: Function 'checkMaterialExist' require 2 arguments.

File: modules/material/material.service.js [hunk 3]
```diff
@@ -32,11 +36,46 @@ class MaterialsService {
...
+    }));
+
+    return new Material({
+      ...rest,
+      additionalPrice: [
+        {
+          currency: 'UAH',
+          value:
+            additionalPrice *
+            Math.round(currency.convertOptions[0].exchangeRate * 100),
```
Comment (dimaPalii) [inline modules/material/material.service.js:70]: Pleace, don't break operator to multiline

File: modules/material/material.service.js (hunk 1 above)
Comment (bandvov) [reply modules/material/material.service.js:42]: fixed

File: modules/material/material.service.js (hunk 2 above)
Comment (bandvov) [reply modules/material/material.service.js:44]: second argument is optional

File: modules/material/material.service.js (hunk 3 above)
Comment (bandvov) [reply modules/material/material.service.js]: fixed

File: __tests__/materials/material.query.test.js [hunk 4]
```diff
@@ -265,7 +264,7 @@ describe('material querries test', () => {
         })
         .catch(e => e);
       const receivedMaterial = res.data.getMaterialById;
-
+      console.log(res);
```
Comment (TorskyM) [inline __tests__/materials/material.query.test.js:267]: Remove `console.log`

File: modules/material/material.service.js [hunk 5]
```diff
@@ -32,11 +36,47 @@ class MaterialsService {
...
+    return new Material({
+      ...rest,
+      additionalPrice: [
+        {
+          currency: 'UAH',
+          value:
+            additionalPrice * Math.round(currency.convertOptions[0].exchangeRate * 100),
+        },
+        {
+          currency: 'USD',
```
Comment (TorskyM) [inline modules/material/material.service.js:74]: Move this `currency` values into config file

File: __tests__/materials/material.query.test.js (hunk 4 above)
Comment (bandvov) [reply __tests__/materials/material.query.test.js:267]: fixed

File: modules/material/material.service.js (hunk 5 above)
Comment (bandvov) [reply modules/material/material.service.js]: fixed

**Review by TorskyM — APPROVED** [review]

**Review by ecl1pseo — APPROVED** [review]

**Review by dimaPalii — APPROVED** [review]

**Pull request merged**